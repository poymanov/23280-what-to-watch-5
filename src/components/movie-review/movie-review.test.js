import React from "react";
import renderer from "react-test-renderer";
import MovieReview from "./movie-review";

const review = {
  id: 1,
  comment: `test`,
  date: `2020-12-20 12:12:12`,
  rating: 5,
  user: {
    id: 1,
    name: `test`,
  }
};

it(`Should MovieReview render correctly`, () => {
  const tree = renderer
    .create(<MovieReview review={review}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
