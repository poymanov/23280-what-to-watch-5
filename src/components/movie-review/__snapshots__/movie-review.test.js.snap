// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieReview render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      test
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        test
      </cite>
      <time
        className="review__date"
        dateTime="2016-12-24"
      >
        December 20, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    5
  </div>
</div>
`;
